memcached.c

```diff
@@ -5016,8 +5016,11 @@ static void process_command(conn *c, char *command) {
 #ifdef TLS
     } else if (ntokens >= 1 && ntokens <= 3 && (strcmp(tokens[COMMAND_TOKEN].value, "refresh_certs") == 0)) {
         set_noreply_maybe(c, tokens, ntokens);
-        refresh_certificates();
-        out_string(c, "OK");
+        if (load_server_certificates()) {
```

Comment:
closer! What I was asking for was that load_server_certificates() returns a string (then use write_and_free here or free() in the initializer), so the user isn't mystified by what caused the error over the protocol.

Reply:
+1, I used the out_string and stack as a return string cannot differentiate between success and a malloc failure without further hand holding.